tests/nn/data_parallel/test_fsdp_memory.py

```diff
@@ -21,15 +21,9 @@
 from fairscale.nn import checkpoint_wrapper
 from fairscale.nn.data_parallel import FullyShardedDataParallel as FSDP
 from fairscale.nn.data_parallel import auto_wrap_bn
+from fairscale.utils import torch_version
 from fairscale.utils.parallel import get_process_group_cached
-from fairscale.utils.testing import (
-    dist_init,
-    dump_all_tensors,
-    skip_if_single_gpu,
-    teardown,
-    temp_files_ctx,
-    torch_version,
-)
+from fairscale.utils.testing import dist_init, dump_all_tensors, skip_if_single_gpu, teardown, temp_files_ctx
```

Comment:
autoformated by isort

Reply:
Yes. Checkout the contribute.md file for all 4 commands to run to autoformat and mypy testing. The CI below runs them too.